plugins/keycloak-backend/src/providers/KeycloakOrgEntityProvider.ts

```diff
@@ -123,34 +126,44 @@ export class KeycloakOrgEntityProvider implements EntityProvider {
   private scheduleFn?: () => Promise<void>;
 
   static fromConfig(
-    configRoot: Config,
-    options: KeycloakOrgEntityProviderOptions,
+    deps: {
+      config: Config;
+      logger: LoggerService;
+    },
+    options: (
+      | { schedule: 'manual' | SchedulerServiceTaskRunner }
```

Comment:
```suggestion
      | { SchedulerServiceTaskRunner }
```

I think we should require a schedule.